ingestion/tests/integration/data_insight/test_web_analytic_events.py

```diff
@@ -114,3 +114,53 @@ def test_web_analytic_events(self):
         )
 
         assert test_event
+
+    def test_delete_web_analytic_event(self):
+        """Test web analytic event deletion"""
+
+        for delta in range(7):
+            tmps = int((datetime.utcnow() - timedelta(days=delta)).timestamp() * 1000)
```

Comment:
Can we call `get_beginning_of_day_timestamp_mill` here?

Reply:
Yep, we should be able to.